core/src/main/scala/akka/persistence/jdbc/state/SequenceNextValUpdater.scala

```diff
@@ -0,0 +1,41 @@
+package akka.persistence.jdbc.state
+
+import slick.jdbc.JdbcProfile
+import slick.dbio.Effect
+import slick.sql.SqlStreamingAction
+
+trait SequenceNextValUpdater {
+  def nextValUpdater(sequenceName: String): String
+  def _getSequenceName(): SqlStreamingAction[Vector[String], String, Effect]
```

Comment:
No underscore in the name since its not private but actual SPI? I guess the entire trait is internal `private[jdbc] ` anyway?

Reply:
done .. the trait is also marked `private[jdbc]`